builders/build/prerequisites.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Prerequisites
+description: Install the basic set of tools and software to set up a local development environment and be able to compile, run and test your ContainerChain.
+---
+
+# Prerequisites {: #prerequisites } 
+
+## Introduction {: #introduction } 
+
+Deploying a ContainerChain through Tanssi is a fairly straightforward step, where the only requirement is to have a valid [chain specification](https://docs.substrate.io/build/chain-spec/){target=_blank} to upload to the Tanssi network and make it go live.
+
+To generate a Substrate chain specification, it is necessary to have a development environment where a substrate node can be compiled, and, to do so, the minimal required software and its installation process will be covered in the next sections of this article.
+
+## Rust {: #rust } 
+
+[Rust](/learn/framework/overview/#rust-programming-language) is a modern, portable, and performant programming language that is the base of the Substrate blockchain development framework.  
+
+To compile the Appchain, the rust compiler *rustc*, and the package manager *cargo* must be installed in the system. 
+
+### Installing Rust via *rustup* {: #install-via-rustup } 
+
+For any system running Linux or MacOS, the following command will do:
+
+=== "Linux & MacOS"
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```     
```

Comment:
I think this looks a bit funny, I'd say either remove the tab or separate Linux and MacOS into two tabs even though it is the same command

Reply:
I also just realized that this isn't a part of the tab, so if you choose to split it out into two, make sure you indent the code snippets four spaces